api/dispatcher.proto

```diff
@@ -126,6 +126,9 @@ message SessionMessage {
 	// Symmetric encryption key distributed by the lead manager. Used by agents
 	// for securing network bootstrapping and communication.
 	repeated EncryptionKey network_bootstrap_keys = 4;
+
+	// Which root certificates to trust
+	bytes RootCA = 5;
```

Comment:
The `NodeCertificateStatusResponse` does not actually contain the root cert bytes, only the TLS certificate.  We could add it to the response, but it seemed less expensive to push the trusted root cert down this way, rather than make the all the nodes have to contact the CA and all renew their TLS certificates a second time just to get the root cert.